Project Mars/MarsQA1/MarsQA-1/Hooks/Hook.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using MarsQA1.Helper;
+using MarsQA1.Specflow_Pages;
+using TechTalk.SpecFlow;
+
+namespace MarsQA1.Hooks
+{
+    [Binding]
+    public class Hooks:Driver
+    {
+        Login_Page login = new Login_Page();
```

Comment:
missing access type please make public and then static if it needs to accessed else where